packages/tasks/src/tasks/image-to-image/data.ts

```diff
@@ -75,33 +75,21 @@ const taskData: TaskDataCustom = {
 	],
 	spaces: [
 		{
-			description: "Image enhancer application for low light.",
-			id: "keras-io/low-light-image-enhancement",
+			description: "Image editing application.",
+			id: "black-forest-labs/FLUX.1-Kontext-Dev",
 		},
 		{
-			description: "Style transfer application.",
-			id: "keras-io/neural-style-transfer",
+			description: "Image relighting application.",
+			id: "kontext-community/kontext-relight",
```

Comment:
How about using https://huggingface.co/spaces/lllyasviel/iclight-v2-vary, which is one of the most visited spaces and brings lots of visibility to ZeroGPU and Pro?